JavaAssignment/src/swatiN/Area.java

```diff
@@ -0,0 +1,38 @@
+package swatiN;
+
+public class Area {
+	static float areaofrect(float len, float br) {
+		float AreaofRect = len * br;
+		System.out.println("Area of rectangle is" + AreaofRect);
+		return AreaofRect;
+	}
+
+	static float perimterofrect(float len, float br) {
```

Comment:
Every methods are returning something, so methods name should be isPerimeterOfRectangle.